opengrok-indexer/src/main/java/org/opengrok/indexer/configuration/RuntimeEnvironment.java

```diff
@@ -1545,30 +1546,30 @@ private void generateProjectRepositoriesMap() throws IOException {
      * If any of the groups contain some projects or repositories already,
      * these get discarded.
      *
-     * @param groups   set of groups to be filled with matching projects
+     * @param map   set of groups to be filled with matching projects
      * @param projects projects to classify
      */
-    public void populateGroups(Set<Group> groups, Set<Project> projects) {
-        if (projects == null || groups == null) {
+    public void populateGroups(Map<String, Group> map, Map<String, Project> projects) {
```

Comment:
Rename `map` to `groups`

Reply:
you dont even need map of groups for this method, just the Set. perhaps using the same trick as with getProjects() would do the trick